docs: example of csharp example serializer (#567)
asyncapi/modelina: docs/languages/Csharp.md

<!-- tocstop -->

## Generate serializer and deserializer functionality

> To generate the data models based on JSON Schema document and validate the serialized data against the schema, use the preset `CSHARP_JSON_SERIALIZER_PRESET`

Reviewer: ```suggestion
Sometimes you want to serialize the data models into JSON, in order to do that use the preset `CSHARP_JSON_SERIALIZER_PRESET`
```
It is not only related to JSON schema, even though you are correct that instantiated data models should always validate accurately against the schema 🙂 If the models are created from a schema of course, but this does not necessarily have to be the case.

Author: Right, I kind of overlooked this. Sorry !
